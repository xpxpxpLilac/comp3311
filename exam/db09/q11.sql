drop view if exists userlike;
create view userlike
as
select u.id as userid, b.style as style, count(*)
from ratings r join beers b on r.beerid = b.id
	join users u on u.id = r.userid
where r.rating > 6
group by userid, style
having count(*)> 2
;

drop view if exists q11;
create view q11
as
select u.name as user, bs.name as style
from userlike ul join users u on ul.userid = u.id
	join beerstyles bs on bs.id = ul.style
;

