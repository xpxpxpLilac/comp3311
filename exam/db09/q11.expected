Vittoria Dubowitz|Altbier
Vittoria Dubowitz|Amber Ale
Vittoria Dubowitz|Pale Ale
Vittoria Dubowitz|Wheat Bier
